selfdrive/car/honda/interface.py

```diff
@@ -273,6 +273,18 @@ def get_params(candidate, fingerprint):
       ret.longitudinalTuning.kiBP = [0., 35.]
       ret.longitudinalTuning.kiV = [0.18, 0.12]
 
+    elif candidate == CAR.FIT:
+      stop_and_go = False
+      ret.mass = 2644. * CV.LB_TO_KG + std_cargo
```

Comment:
```suggestion
      ret.mass = 2644. * CV.LB_TO_KG + STD_CARGO_KG
```
It's now STD_CARGO_KG on 0.6.x